File: Bedu/DesarrolloMobileJava/Sesion7/Sesion7/app/src/main/java/xyz/eliothmonroy/sesion7/reto1/utils/RetrofitUtils.java
package xyz.eliothmonroy.sesion7.reto1.utils;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class RetrofitUtils {

    public static Retrofit getRetrofitInstance(){
        return new Retrofit.Builder().baseUrl(Constants.BASE_URL_API_USERS)
                .addConverterFactory(GsonConverterFactory.create()).build();
    }

}

File: Bedu/KotlinFundamentals/Sesion6/retrofitDemo/app/src/main/java/xyz/eliothmonroy/retrofitdemo/reto1/api/RetrofitUtils.kt
package xyz.eliothmonroy.retrofitdemo.reto1.api

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

object RetrofitUtils {

    fun getRetofitInstance():Retrofit{
        return Retrofit.Builder().baseUrl(CONSTANTS.BASE_URL).addConverterFactory(GsonConverterFactory.create()).build()
    }

}